.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Report a bug or a performance issue
+about: Use this template to report unexpected behavior
+title: ''
+labels: ''
+assignees: ''
+---
+
+# Summary
+Provide a short summary of the issue. Sections below provide guidance on what
+factors are considered important to reproduce an issue.
+
+# Version
+Report oneMKL version and githash.
+
+# Environment
+oneMKL works with multiple HW and backend libraries and also depending on the compiler and build environment. Include
+the following information to help reproduce the issue:
+* HW you use
+* Backend library version
+* OS name and version 
+* Compiler version 
+* CMake output log
+
```

Comment:
Does it make sense to ask for the whole build directory?

Reply:
I don't think it is needed always and might be overkill to ask to provide it. Log should be sufficient and if we need more information, we can always ask